elasticsearch/helpers/__init__.py

```diff
@@ -254,6 +256,13 @@ def scan(client, query=None, scroll='5m', raise_on_error=True, preserve_order=Fa
         cause the scroll to paginate with preserving the order. Note that this
         can be an extremely expensive operation and can easily lead to
         unpredictable results, use with caution.
+    :arg global_kwargs: additional kwargs to be passed to both
+        :meth:`~elasticsearch.Elasticsearch.search` and to
+        :meth:`~elasticsearch.Elasticsearch.scroll`
+    :arg search_kwargs: additional kwargs to be passed to
+        :meth:`~elasticsearch.Elasticsearch.search`
+    :arg scroll_kwargs: additional kwargs to be passed to
+        :meth:`~elasticsearch.Elasticsearch.scroll`
 
     Any additional keyword arguments will be passed to the initial
     :meth:`~elasticsearch.Elasticsearch.search` call::
```

Comment:
If we have this here we should remove the `search_kwargs` and just use `kwargs` there, it would also be backwards compatible just like the current solution and simpler.